blog/docs/releases/announcing-knative-v1-4-release.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "v1.4 release"
+linkTitle: "v1.4 release"
+Author: "[Nader Ziada](https://twitter.com/nziada)"
+Author handle: https://github.com/nader-ziada
+date: 2022-04-20
+description: "Knative v1.4 release announcement"
+type: "blog"
+---
+
+
+## Announcing Knative 1.4 Release
+
+A new version of Knative is now available across multiple components.
+
+Follow the instructions in
+[Installing Knative](https://knative.dev/docs/install/) to install the components you require.
+
+## Table of Contents
+- [Highlights](#highlights)
+- [Serving v1.4](#serving-v1.4-NOSPACES)
+- [Eventing v1.4](#eventing-v1.4-NOSPACES)
+- [Eventing Extensions](#eventing-extensions)
+    - [Apache Kafka Broker v1.4](#apache-kafka-broker-v1.4-NOSPACES)
+    - [RabbitMQ Broker and Source v1.4](#rabbitmq-broker-and-source-v1.4-NOSPACES)
+- `kn` [CLI v1.4](#client-v1.4-NOSPACES)
+- [Knative Operator v1.4](#operator-v1.4-NOSPACES)
+- [Thank you contributors](#thank-you-contributors)
+
+
+## Highlights
+
+- Our minimum supported Kubernetes version is now 1.22
+
+
+## Serving v1.4
+
+<!-- Original notes are here: https://github.com/knative/serving/releases/tag/knative-v1.4.0 -->
+
+🚨 Breaking or Notable
+ - Our minimum supported Kubernetes version is now 1.22 (#12753, @dprotaso)
```

Comment:
All entries like `(#12753, @dprotaso)` need to only have the PR number, remove the user id it should be moved to the end of the document. something like `([#12753](https://github.com/knative/serving/pull/12753))`

Reply:
updated all RPs links